### PR TITLE
Add `rgh-linkify-yolo-issues` feature 

### LINKED ISSUE #6258: Linkify issues on yolo
### Description

We have issues on this page, but they're not clickable. This would be a quick meta feature, probably named `rgh-linkify-yolo-issues`

### Screenshot

Screenshot 31

### Example URLs

https://github.com/refined-github/yolo/blob/main/broken-features.csv

### PR DESCRIPTION
**Pull request by hemantsoni42**
this code will replace the original text content with an anchor element
In previous pull request the file name is rgh-linkify-yolo-issues.js which fails in some checks that why in this the file name is renamed as rgh-linkify-yolo-issues.tsx 
Closes #6258

### CODE REVIEW

**Review by fregante — CHANGES_REQUESTED** [review]
Thank you for the PR, unfortunately this is missing a lot of work for now.

Please refer to the existing features, there's a complete documentation in the contributing.md file. Also there are several issues in the code:

- I suggested using JSX, `wrap` and `select`, instead of `createElement` and `appendChild` and `querySelectorAll`. Please copy the snippet of code I gave you because it was 90% valid
- the `td:nth-child` selector is way too broad, it must be limited to the specific table. This is the 10% that was missing.
- This file is missing `features.add` and all the logic around it, look at `rgh-linkify-features` for example
- Make sure you build and run the extension locally before submitting. For example here the tests are failing and this file is not loaded at all because it's not imported in `refined-github.ts

**Comment by hemantsoni42** [comment]
Thank you sir for all your support and guidance.
Currently this newly added file is not showing any error in local machine. If I will not satisfies your goal please let me know and thank you once again.

File: rgh-linkify-yolo-issues.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from "dom-chef";
+import * as pageDetect from "github-url-detection";
+import { wrap } from "../helpers/dom-utils";
+import features from "../feature-manager";
+import featureLink from "../helpers/feature-link";
+import { getNewFeatureName } from "../options-storage";
+import { isAnyRefinedGitHubRepo } from "../github-helpers";
+import observe from "../helpers/selector-observer";
+function init(): void {
+	for (const issueCell of select.all("table-selector td:nth-child(2)")) {
```
Comment (fregante) [inline rgh-linkify-yolo-issues.tsx:10]: What's `table-selector`? This PR hasn't been tested. Test before marking as "ready"

File: rgh-linkify-yolo-issues.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+features.add({
+	include: [
+		pageDetect.hasComments,
+		pageDetect.isReleasesOrTags,
+		pageDetect.isCommitList,
+		pageDetect.isSingleCommit,
+		pageDetect.isRepoWiki,
+		pageDetect.isPR,
+		pageDetect.isIssue,
```
Comment (fregante) [inline rgh-linkify-yolo-issues.tsx:28]: All completely unrelated. Please don't copy-paste blindly, each line in the file must make sense

File: rgh-linkify-yolo-issues.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	],
+	asyncDomReady: false,
+	init,
+	asLongAs: isAnyRefinedGitHubRepo,
+	url: import.meta.url,
```
Comment (fregante) [inline rgh-linkify-yolo-issues.tsx:33]: Completely invalid usage

**Review by fregante — CHANGES_REQUESTED** [review]
Please run the extension locally, ensure that `npm test` completes successfully and _only then_ push your changes and ask for a review. As is, this does not work.

Make sure it works and include a screenshot on the next push please.

**Comment by fregante** [comment]
> this file is not loaded at all because it's not imported in `refined-github.ts`

**Comment by fregante** [comment]
Sorry I don’t think you can submit further changes here

**Pull request closed without merging**